frontend/pages/admin/UserManagementPage/components/UserForm/_styles.scss

```diff
@@ -100,10 +108,23 @@
     }
   }
 
-  .sublabel {
+  .sublabel, .sublabel-nosso {
     margin: 0px;
   }
 
+  .sublabel-nosso {
+    font-size: 0.75rem;
+    .link-button {
+      display: inline-flex;
+      font-size: 0.75rem;
+      padding: 0;
+      border: 0;
+      cursor: pointer;
+      background: transparent;
+      color: $core-vibrant-blue;
+    }
```

Comment:
This code tries to make a button look like a link. Perhaps we already have such a mixin?

Reply:
the closest I could find is to remove this and add the following classes to the button: `button button--text-link`, but might need additional tweaking (for the font size)

Reply:
Yes, seems to work.